src/types/index.ts

```diff
@@ -124,6 +124,7 @@ export type SavedItemUpsertInput = {
 };
 
 export const defaultPage = 30;
+export const maxPageSize = 100;
```

Comment:
should these be coming from a `config` file instead of a `types` file?

Reply:
moved to config